velox/dwio/dwrf/common/FileMetadata.h

```diff
@@ -405,11 +405,12 @@ class FooterWrapper : public ProtoWrapperBase {
 
   bool hasRowIndexStride() const {
     return format_ == DwrfFormat::kDwrf ? dwrfPtr()->has_rowindexstride()
-                                        : false;
+                                        : orcPtr()->has_rowindexstride();
```

Comment:
Thanks for the fix.  Can we add a unit test?

Reply:
Yes, I've already added one.